### PR TITLE
Utilize n_top_pmts in peaklet processing

### PR DESCRIPTION
**Pull request by DCichon**
Adds a parameter to peaklet processing in order to be able to build the summed waveform over a single array only (see https://github.com/AxFoundation/strax/pull/565 for more details).

### CODE REVIEW

**Review by jmosbacher — CHANGES_REQUESTED** [review]
One very important thing I dont see here is bumping the plugin version. Since the plugin will produce different data, its crucial that the version is changes so that the lineage changes and data is reprocessed with the new plugin even if it was processed previously with the old one.

**Comment by jmosbacher** [comment]
Updated to use URLConfigs.

**Review by FaroutYLq — APPROVED** [review]
Tested and it is working after e20104f673dcfe8d7448ec2513c554ef4c723ce6

File: straxen/plugins/peaklet_processing.py [hunk 1]
```diff
@@ -236,7 +240,8 @@ def compute(self, records, start, end):
             peak_list = peak_saturation_correction(
                 r, rlinks, peaklets, hitlets, self.to_pe,
                 reference_length=self.saturation_reference_length,
-                min_reference_length=self.saturation_min_reference_length)
+                min_reference_length=self.saturation_min_reference_length,
+                n_top_channels=n_top_pmts)
```
[the remark this answers is not in the document]
Comment (jmosbacher) [reply straxen/plugins/peaklet_processing.py]: Yesh, looks like a typo :)

**Comment by JoranAngevaare** [comment]
I'll remake this PR in a sec

**Comment by JoranAngevaare** [comment]
I also tested that the data produced yields the same for:
1. current master
2. the version that Lanqing tested (https://github.com/XENONnT/straxen/pull/721)
3. the updates that I made to that later (https://github.com/XENONnT/straxen/pull/1099)

The notebook for the test is here:
https://github.com/XENONnT/analysiscode/pull/98

**Pull request closed without merging**